utils/cluster.js

```diff
@@ -15,3 +16,13 @@ export function isHarvesterCluster(mgmtCluster) {
 
   return provider === VIRTUAL_HARVESTER_PROVIDER;
 }
+
+export function isHarvesterSatisfiesVersion(version = '') {
```

Comment:
This should be called something more generic, as it's applied in all cases

Reply:
This looks like it's changed the natural flow for non-harvester clusters. Could you explain the change?

Reply:
Create a Harvester cluster requires RKE version >= `v1.21.4+rke2r4`. So I need to init k8s version when select create a harvester cluster

```js
      if (this.isHarvesterDriver && showRke2) {
        this.setHarvesterK8sDefaultVersion();
      } else {
        set(this.value.spec, 'kubernetesVersion', option.value);
      }
```

Reply:
Sorry, i hadn't spotted this fn was only ever called in Harvester world